src/components/openHouse/page.tsx

```
@@ -0,0 +1,415 @@
+  type,
+  orgsData,
+  collapsed,
+  onToggle,
+  selectedBooth,
+  handleBoothSelect,
+}: BoothSectionProps) => (
+  <div className="flex flex-col gap-2">
+    <h3
+      className="flex cursor-pointer items-center justify-between rounded-xl border border-navy-100 bg-gradient-to-br from-navy-50 to-surface-100 px-4 py-3 text-lg font-semibold text-navy-800 select-none transition-all hover:from-navy-100 hover:to-surface-150"
+      onClick={onToggle}
+    >
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)